apps/wallet/src/background/keyring/VaultStorage.ts

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { randomBytes } from '@noble/hashes/utils';
+import Browser from 'webextension-polyfill';
+
+import { Vault } from './Vault';
+import { getRandomEntropy, toEntropy } from '_shared/utils/bip39';
+
+import type { StoredData } from './Vault';
+import type { Storage } from 'webextension-polyfill';
+
+export const IS_SESSION_STORAGE_SUPPORTED = 'chrome' in global;
+const SESSION_STORAGE: Storage.LocalStorageArea | null =
+    // @ts-expect-error chrome
+    IS_SESSION_STORAGE_SUPPORTED ? global.chrome.storage.session : null;
+const LOCAL_STORAGE = Browser.storage.local;
+
+// we use this password + a random one for each time we store the encrypted
+// vault to session storage
+const PASSWORD =
+    process.env.WALLET_KEYRING_PASSWORD ||
+    '344c6f7d04a65c24f35f5c710b0e91e2f2e2f88c038562622d5602019b937bc2c2aa2821e65cc94775fe5acf2fee240d38f1abbbe00b0e6682646a4ce10e908e';
+const VAULT_KEY = 'vault';
+const EPHEMERAL_PASSWORD_KEY = '244e4b24e667ebf';
```

Comment:
What are these values?

Reply:
Random chars, in case we want to change them and avoid using password/vault